lsmb-request.pl

```diff
@@ -19,6 +19,7 @@ =head1 COPYRIGHT
 =cut
 
 package LedgerSMB::Handler;
+use lib '.';
```

Comment:
http://search.cpan.org/~nwclark/perl-5.8.7/lib/FindBin.pm has some docs for FindBin, but again, it is *not* working for Debian at present, so my guess is they broke something more significant.....